src/detect.c

```diff
@@ -1613,6 +1613,46 @@ static inline bool ApplyAcceptToPacket(
     return false;
 }
 
+/** \internal
+ * \retval bool true: break_out_of_app_filter, false: don't break out */
+static bool ApplyAccept(Packet *p, const uint8_t flow_flags, const Signature *s,
+        DetectTransaction *tx, const int tx_end_state, const bool fw_next_progress_missing,
+        bool *tx_fw_verdict, bool *skip_fw_hook, uint8_t *skip_before_progress)
```

Comment:
as a follow up, thinking about having a little dedicated struct to hold the fw flow control vars to avoid this long list of arg ptrs